resources/server.rb

```diff
@@ -74,16 +74,19 @@
     end
     notifies :delete, "file[#{home_dir}/.sdm/roles]", :immediately
     not_if do
-      tester = Mixlib::ShellOut.new(
+      # short-circuits to prevent reaching out to strongdm
+      return false unless ::File.exist?("#{home_dir}/.ssh/authorized_keys")
+      return false if ::File.exist?("#{home_dir}/.ssh/authorized_keys") && ::File.empty?("#{home_dir}/.ssh/authorized_keys")
```

Comment:
Checking if the file exists here is redundant, since the previous line would exit the block if it doesn't exist.

Reply:
Copy pasta